addons/l10n_it_stock_ddt/data/l10n_it_ddt_template.xml

```diff
@@ -5,7 +5,7 @@
                 <t t-if="ddt_dict and not record.l10n_it_ddt_id">
                     <t t-foreach="ddt_dict" t-as="picking">
                         <DatiDDT>
-                            <NumeroDDT t-esc="picking.l10n_it_ddt_number"/>
+                            <NumeroDDT t-esc="format_alphanumeric(picking.l10n_it_ddt_number[-20:])"/>
```

Comment:
https://github.com/odoo/odoo/blob/5a2556e35c87cc363ac60a085a39f88efe2d21fe/addons/l10n_it_stock_ddt/models/stock_picking.py#L62-L70

5 (max warehouse code) + 1 (`/`) + `?` (sequence code) + 4 (`/DDT`) + 5 (padding), leaves "only" 6 chars for the sequence code